### ISSUE TITLE
Powder Relics Removing Items?

### ISSUE THREAD

**Issue by Kronomancer** [Wynncraft/Issues#8315]
I'm not sure whether to post this here or in the feedback section, so I made both

So ye
I was on my way to Rodoroc, and I decided to just enter a relic and get a powder, because why not. The thing happened, I got a T4 Air, and it was placed in my second hotbar slot, completely removing my Judas from the face of Gavel, not even dropping it on the ground.

I have no idea if that has to do with weapon skin item or powder item or whatever, but this seems like a big problem if it keeps happening to other people even though no one uses relics

P.S. 
I know that there is no refund policy and how the fuck someone even supposed to go me an untradable item back if there wasn't this policy?

**Comment by Kronomancer** [comment]
Anyone?

**Comment by Hallie2K17** [comment]
Try switching servers that happend to me with the "Grave Mistake" Quest i got Telported and  Due to server Crash + Lag i was Frozen at The old Church of the Undead in a Invisible box i did /kill+ switched servers and still the quest remains and i didint complete it from that time

**Comment by Hallie2K17** [comment]
Is The item Lored with "Quest Item" or not? if not then no refund and if it is Contact a admin or Mod to Help u, Hope i helped u at something :D

**Comment by Kronomancer** [comment]
No, the item is Judas, a thunder wand from Corrupted Sand Swept Tomb. And I'm talking about a weird glitch made it be replaced with powder when I touched the relic.


I'm not asking for refunds btw, I'm just pointing out the problem.
